src/dsp_tools/commands/xmlupload/make_rdf_graph/make_file_value.py

```diff
@@ -22,28 +22,29 @@
 from dsp_tools.models.exceptions import BaseError
 
 
-def make_iiif_uri_value_graph(iiif_uri: AbstractFileValue, res_bn: BNode) -> tuple[Graph, URIRef]:
+def make_iiif_uri_value_graph(iiif_uri: AbstractFileValue, res_node: BNode | URIRef) -> Graph:
     """
     Creates a graph with the IIIF-URI Link
 
     Args:
         iiif_uri: Information about the IIIF URI
-        res_bn: Blank-node of the resource
+        res_node: Node of the resource
 
     Returns:
         Graph with the IIIF-URI Value
     """
-    g = _make_abstract_file_value_graph(iiif_uri, IIIF_URI_VALUE, res_bn, KNORA_API.stillImageFileValueHasExternalUrl)
-    return g, KNORA_API.hasStillImageFileValue
+    return _make_abstract_file_value_graph(
```

Comment:
The propery name was needed for the framing when we only had the values. Now that we have the resource we can frame it in a more generalised way.